src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/PhysicalAddress.cs

```diff
@@ -170,6 +226,10 @@ public static PhysicalAddress Parse(string address)
                 {
                     throw new FormatException(SR.Format(SR.net_bad_mac_address, address));
                 }
+                if ((hasPeriods || hasColons) && validCount >= 4)
+                {
```

Comment:
Do you have test cases that hit each of these new exceptions you’re adding to the implementation?

Reply:
No, but I can add them